index.html

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html>
     <head>
+        <!-- Prompt a message in the browser if users disabled JS -->
+        <noscript>Your browser does not support JavaScript!</noscript>
```

Comment:
should we translate this?

Reply:
We can't localize texts here as there can be only one `/index.html` file (in root).

Reply:
Since the service_worker's start_url has been set to `<lang>/app/index.html`, it doesn't need any more to have this tag here in this page. it will be moved to `src/templates/app_2/head.jsx` since there is accessibility to the localization method there.